ejb-txn-remote-call/README.adoc

```diff
@@ -0,0 +1,1092 @@
+
+[#_running_on_kubernetes_run_with_productname_operator]
+==== Running on Kubernetes: run with {productName} Operator
+
+You deploy the {productName} Operator via Kubernetes `Deployment` object.
+The {productName} Operator listens to changes at other Kubernetes objects
+of the type `CustomerResource`.
+The WildFly Operator manages `CustomerResource` of kind `WildFlyServer`.
+
+The WildFly Operator can be found at https://quay.io[Quay.io]
+repository at https://quay.io/repository/wildfly/wildfly-operator
+with source code at https://github.com/wildfly/wildfly-operator.
+
+You can find the Operator's `Deployment` definition in the YAML descriptor
+at https://github.com/wildfly/wildfly-operator/blob/master/deploy/operator.yaml[WildFly Operator Github repository].
+
+For the deployment to work correctly a https://github.com/wildfly/wildfly-operator/blob/master/deploy/service_account.yaml[service account],
+https://github.com/wildfly/wildfly-operator/blob/master/deploy/role.yaml[a role] and
+https://github.com/wildfly/wildfly-operator/blob/master/deploy/role_binding.yaml[a role binding] have to be created
+in the Kubernetes cluster.
+
+The follow-up step is creation of https://github.com/wildfly/wildfly-operator/blob/master/deploy/crds/wildfly_v1alpha1_wildflyserver_crd.yaml[`CustomResourceDefinition`]
+(abbreviated as *CRD*) which defines what capabilities the Operator provides and which things may be configured for the `WildFlyServer` `CustomerResource`.
+
+[NOTE]
+====
+If you clone the https://github.com/wildfly/wildfly-operator[WildFly Operator GitHub repository] to your
+local disk you may use the prepared script https://github.com/wildfly/wildfly-operator/blob/master/build/run-minikube.sh[build/run-minikube.sh]
+for that purpose.
+
+[source,sh,options="nowrap"]
+----
+git clone https://github.com/wildfly/wildfly-operator
+./wildfly-operator/build/run-minikube.sh
+----
+====
+
+The quickstart uses clustering.
+The {productName} clustering works with https://github.com/jgroups-extras/jgroups-kubernetes[jgroups `KUBE_PING`]
+protocol. This protocol requires having permission to list all available pods in scope of the `namespace`.
+The `default` `ServiceAccount` does not have such permissions.
+For development purposes it is possible to use
+https://github.com/wildfly/wildfly-operator/blob/master/examples/clustering/crds/role_binding.yaml[`RoleBinding` definition from WildFly Operator repository].
+The definition permits for the deployments to view details information about any Kubernetes object
+inside of the current `namespace`.
+
+[source,sh,options="nowrap"]
+----
+kubectl create -f https://raw.githubusercontent.com/wildfly/wildfly-operator/master/examples/clustering/crds/role_binding.yaml
+----
+
+When all this is set up, and the {productName} Operator `Pod` is running we may prepare a definition
+of the `CustomerResource` which makes the application deployed.
```

Comment:
```suggestion
of the `CustomResource`, which makes the application deployed.
```
I'm not sure what edit to recommend for `makes the application deployed`.  I suspect that the `CustomResource` does not actually do the deployment.  Perhaps "is used to deploy the application"?